setup.py

```diff
@@ -256,4 +257,5 @@ def can_import(module_name):
     ext_modules=EXTENSIONS,
     include_package_data=True,  # done via MANIFEST.in under setuptools
     install_requires=REQUIRES,
+    python_requires=">={}".format(".".join(str(x) for x in PYVER)),
```

Comment:
From a style point of view, we're avoiding the format method and starting to use f-strings in the main code. However, setup.py is special as it is very helpful for it to work on older Python versions and run enough to give the version warning. So how about:

```python
python_requires=">=%i.%s" % PYVER,
```